tests/CourseDashboardTests.cpp

```diff
@@ -1,8 +1,35 @@
 #include <gtest/gtest.h>
 #include "CourseDashboard.hpp"
+#include "User.hpp"
 
+class CourseDashboardTests : public ::testing::Test
+{
+protected:
+    CourseDashboard* testCdb;
+    User* testU1;
```

Comment:
why pointers, not normal values?

Reply:
I thought this variable will be more powerful:), but without deeper sense. Cause we don't use polymorphism yet.

Reply:
Currently there is no any pointers in tests.